WordPressKit/WordPressComRestApi.swift

```diff
@@ -510,6 +544,19 @@ extension WordPressComRestApi {
     }
 }
 
+// MARK: - Constants
+
+private extension WordPressComRestApi {
+
+    enum Constants {
+        static let buildRequestError = NSError(domain: String(describing: WordPressComRestApiError.self),
+                                               code: WordPressComRestApiError.requestSerializationFailed.rawValue,
+                                               userInfo: [NSLocalizedDescriptionKey: NSLocalizedString("Failed to serialize request to the REST API.",
+                                                                                                       comment: "Error message to show when wrong URL format is used to access the REST API")])
```

Comment:
This error was already there, I just put it in there to avoid code duplication. My take is that, since the error is provided by the API, the consumer can do whatever they want with it, including adding it to an alert message, so I'd leave it as it is.